webtransport/client.js

```diff
@@ -129,17 +129,17 @@ async function acceptUnidirectionalStreams(transport) {
   }
 }
 
-async function readFromIncomingStream(stream, number) {
+async function readFromIncomingStream(readable, number) {
   let decoder = new TextDecoderStream('utf-8');
```

Comment:
`decoder` is a TextDecoderStream but you handle it as if it is a TextDecoder.

Reply:
```
let reader = readable.pipeThrough(decoder).getReader();
// ...
let data = value;
```

Reply:
Fixed.